### ISSUE TITLE
Chapter III Section 5.b.1.a.3

### ISSUE THREAD

**Issue by Nazeeefa** [elixir-no-nels/DSW-ELIXIR-NO-feedback#145]
Under section 5 "Will you be acquiring data using measurement equipment?", there is a sub-section "5.b.1.a.3 Instruments used for data collection".

The section asks for information about instrument name and instrument description.
It may be more helpful if description of this section mentions that, in addition to "what technical instruments you are using to collect the data", summarise purpose of technical instrument you are using to collect the data.

I understand we provide definition (for exemplary DMPs) in instrument descriptions with external link. However, for DM Plan, it's more helpful (both for researcher and even grant reviewer) if description also includes brief mention of the purpose of an instrument in description.

Example (Instrument Description): "The PacBio Sequel Systems are powered by Single Molecule, Real-Time (SMRT) Sequencing technology and deliver highly accurate long reads. More information about PacBio Sequel system can be found at: https://www.pacb.com/products-and-services/sequel-system/. For our research, PacBio Sequel IIe will be used to 123 generate _de-novo_ genome assemblies of xyz organism. We chose PacBio Sequel IIe as it can help us to in xyz ways"

Happy to know your thoughts/views :)

**Comment by korbinib** [comment]
Agreed in principle. (At a moment a grant reviewer would not see this.) If we remodel the whole question we would have to adapt the export templates also accordingly.
